catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/FrameworkProperties.java

```diff
@@ -57,7 +57,7 @@
 
   private List<CatalogProvider> catalogProviders = new ArrayList<>();
 
-  private Map<String, CatalogStore> catalogStoresMap = new HashMap<>();
+  private Collection<CatalogStore> catalogStores = Collections.emptyList();
```

Comment:
❓ Should we use a thread-safe collection here?

Reply:
see [above](https://github.com/codice/ddf/pull/4250#discussion_r256468534)